pkg/operator/status.go

```diff
@@ -385,6 +386,123 @@ func (optr *Operator) syncMetrics() error {
 	return nil
 }
 
+func (optr *Operator) syncClusterFleetEvaluation() error {
+	co, err := optr.fetchClusterOperator()
+	if err != nil {
+		return err
+	}
+	if co == nil {
+		return nil
+	}
+
+	unexpectedEvaluations, err := optr.generateClusterFleetEvaluations()
+	if err != nil {
+		return err
+	}
+
+	var coStatus configv1.ClusterOperatorStatusCondition
+	if len(unexpectedEvaluations) > 0 {
+		coStatus = configv1.ClusterOperatorStatusCondition{
+			Type:   configv1.EvaluationConditionsDetected,
+			Status: configv1.ConditionTrue,
+			Reason: evaluationToString(unexpectedEvaluations),
+		}
+	} else {
+		coStatus = configv1.ClusterOperatorStatusCondition{
+			Type:   configv1.EvaluationConditionsDetected,
+			Status: configv1.ConditionFalse,
+			Reason: asExpectedReason,
+		}
+	}
+
+	return optr.updateStatus(co, coStatus)
+}
+
+func evaluationToString(e []string) string {
+	if len(e) == 0 {
+		return ""
+	}
+	return strings.Join(e, "::")
+}
+
+func (optr *Operator) generateClusterFleetEvaluations() ([]string, error) {
```

Comment:
This should be used to flag items we're considering adding upgradeable=false for.  For general collection of interesting data, regular telemetry is appropriate.  The reasoning is that to do upgradeable=false, this code must be written, but for general collection it's unnecessary clutter.

Reply:
Added a comment.